ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -411,7 +411,7 @@ class ActionBar extends React.PureComponent {
               <Button
                 id="pin-job-btn"
                 title="Add this job to the pinboard"
-                className="btn icon-blue"
+                className="actionbar-nav-btn btn icon-blue bg-transparent border-0"
```

Comment:
I wonder if it'd be worth eliminating the custom blue and cyan colors and replacing them with bootstrap colors, such as 'primary' and 'info'. Then if you used the reactstrap Button element and passed in the color prop, I think it'd handle the hover and focus style attributes that `icon-blue` is used for. Although, from the name of the style class perhaps it's a bit more complicated since it seems like its mostly used for icons, not buttons?

Reply:
Yeah, it might.  But it would require some code work for the hover on these buttons.  These only show colors on hover/focus/active.  So I can't just replace the class with ``text-info``.

Reply:
Actually, I think I may be off on my reply, sorry...  :)  Let me check that out further...

Reply:
yeah, so I use a reactstrap component for all Buttons and it has all of the hover and focus effects. You just pass a color prop :)

Reply:
Looks like the hover for those colors only changes the background.  But this is a light button on a black background and we want hover to change the CSS ``color`` prop.  So I think I'll leave this as-is for now, if that's cool.